docs/goals.md

```diff
@@ -1,16 +1,75 @@
 # Goals and Non-Goals
 
-This document defines the purpose of the Message Format Working Group (MFWG)
-and informs the decisions about the scope and the priorities of its efforts.
+This document contains the charter for the MessageFormat Working Group (MFWG)
+and informs decisions about the scope and priority of its efforts.
 
-## Goals
+## Charter
 
-The primary task of the MFWG is to develop an industry standard for the
-representation of localizable dynamic message strings. A **_dynamic message
-string_** is a string whose content changes due to the value of or insertion
+A **_dynamic message string_** is a string whose content changes due to the value of or insertion
 of some data value or values.
 
-The design goals are listed below.
+The _Unicode MessageFormat Standard_ is an industry standard for the representation
+of localizable _dynamic message strings_.
+
+The MessageFormat Working Group maintains and extends the Unicode MessageFormat Standard,
+provides documentation;
+encourages implementation, including the development of tools and best practices;
+manages default and Unicode-defined function sets;
+and provides for interoperability with other standards.
+
+The MessageFormat Working Group is a working group of the CLDR-TC.
+
+## Goals
+
+- Maintain and develop the [messageformat.unicode.org] site,
+  maintaining a high bar (as a model for other Unicode documentation),
+  including at least 
+  a user guide,
+  and a migration guide from other formats (including ICU MessageFormat).
+- Encourage adoption of Unicode MessageFormat by:
+  providing implementation support materials;
+  creating tests;
+  linking to success stories, tools, and implementations;
+  supporting or hosting workshops or supporting presentations by members;
+  and supporting adoption by encapsulating standards (such as ECMA-402, MessageResource, etc.)
+- Support migration and adoption by publishing as Stable in the default function set
+  the functions and options needed to provide compatibility with ICU MessageFormat ("MF1").
+  Note that certain features of MF1 have been deliberately excluded from this goal
+  because they are in conflict with the design goals of Unicode MessageFormat,
+  because they are specialized features unique to MF1 (which ICU is free to implement),
+  or because they might be superseded by newer specifications:
+  - ChoiceFormat (an anti-pattern)
+  - Date/time picture strings (an anti-pattern)
+  - Date/time and number skeletons (specialized functionality, potentially superseded by semantic skeletons)
+  - Nested selection or partial message selection (an anti-pattern)
+  - `spellout` and `duration` formats (specialized functionality)
+  - `ordinal` _formatting_ (specialized functionality, incompetely specified in CLDR)
```

Comment:
```suggestion
  - `ordinal` _formatting_ (specialized functionality)
```
Ordinal formatting is accomplished by having the ordinal categories, and inflections (in where needed). Both are _defined_ in CLDR.
What is lacking is not the definition, but rather that some languages lack data.